### PR TITLE
Add auditlog/cleanup management endpoint #294

### PR DESCRIPTION
**Pull request by pmi**

### CODE REVIEW

File: internal/app/commands/auditlog/cleanup.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+func ensureAgeParam(c *cli.Context) string {
+	force := common.IsForceMode(c)
+	return util.PromptString("Enter age threshold in ISO-8601 based duration format (PnDTnHnMn.nS):", c.String("age"), "", func(val interface{}) error {
+		str := val.(string)
+		if len(strings.TrimSpace(str)) == 0 {
+			if force {
+				fmt.Fprintln(os.Stderr, "Age threshold can not be empty in non-interactive mode.")
+				os.Exit(1)
```
Comment (gbbirkisson) [inline internal/app/commands/auditlog/cleanup.go:54]: I would rather return the error and also return it in the calling function. That way all errors are handled by the same piece of code. Calling os.Exit(1) all over the place seems a bit dirty.

Or is this is how its done all over the CLI?

Comment (pmi) [reply internal/app/commands/auditlog/cleanup.go:54]: Well, force flag was introduced later and it's how I did it to keep changes down to min everywhere :\
But that doesn't mean we should stick to that with new code, right ? ;)

Comment (gbbirkisson) [reply internal/app/commands/auditlog/cleanup.go:54]: You can propagate the errors upwards so they get handled by this code: https://github.com/enonic/cli-enonic/blob/5e78c2770a64134114344aac55b1b259c5dd7880/internal/app/cli.go#L37-L39

In fact that is what I did in the cloud part of the cli: https://github.com/enonic/cli-enonic/blob/5e78c2770a64134114344aac55b1b259c5dd7880/internal/app/commands/cloud/login.go#L24-L33

I would consider that best practice. There is one big problem though and that is that if we propagate errors up, the cli only logs the error and does not exit with an exit code: https://github.com/enonic/cli-enonic/blob/5e78c2770a64134114344aac55b1b259c5dd7880/internal/app/cli.go#L37-L39

Comment (pmi) [reply internal/app/commands/auditlog/cleanup.go:54]: Agree with all of the above, but I just realized there's a reason we can't propagate error here: this code is inside of an input validator function.
Validator error is cosidered an input error and user is asked to re-enter value. Null is considered a passing check.
Aaand the exit code is very important for CLI scripting, so we most definitely need it in place.

Comment (alansemenov) [reply internal/app/commands/auditlog/cleanup.go:54]: @gbbirkisson do we leave it like this?

Comment (gbbirkisson) [reply internal/app/commands/auditlog/cleanup.go:54]: I guess so

**Pull request merged**